bitbot/reminder.go

```diff
@@ -0,0 +1,287 @@
+package bitbot
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/whyrusleeping/hellabot"
+	log "gopkg.in/inconshreveable/log15.v2"
+)
+
+//nolint gochecknoglobals
+var (
+	location   *time.Location
+	timeFormat string
+)
+
+// ReminderEvent : The Gorm struct that represents an event in the DB.
+type ReminderEvent struct {
+	ID          int `gorm:"unique;AUTO_INCREMENT;PRIMARY_KEY"`
+	Channel     string
+	Author      string
+	Description string
+	People      string
+	Time        time.Time
+}
+
+var ReminderTrigger = NamedTrigger{ //nolint:gochecknoglobals,golint
+	ID:   "reminder",
+	Help: "Set up events and remind them to concerned people. Usage: !remind list|time|add|remove|join|part",
+	Condition: func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "PRIVMSG" && strings.HasPrefix(m.Trailing, "!remind")
+	},
+	Action: func(irc *hbot.Bot, m *hbot.Message) bool {
+		timeFormat = "2006-01-02 15:04"
+
+		var err error
+		location, err = time.LoadLocation("UTC")
```

Comment:
why are we declaring location globally and then setting it here?

Reply:
It should be in the init, I forgot about it.